js/column.js

```diff
@@ -324,8 +421,13 @@ ReferenceColumn.prototype = {
 };
 
 /**
- * TODO aggregate is currently ignored since it's not being used.
- * We should add suppoort for aggregate to pseudo-columns later
+ * If you want to create an object of this type, use the `module._createPseudoColumn` method.
```

Comment:
I know there isn't a way to make anything `private`, but we could preface `private` constructors (maybe using `_` still) so that it is more clear looking at the code.

Reply:
All these prototypes are internal and are not available on the `ERMrest` module. It's true about almost all of the prototypes that we have in ermestjs (`Reference`, `Page`, `Tuple`, `Column`, etc.). So the notaion of making it `private` wouldn't do much in this case since it's always being used within ermrestjs. This is more of a note that if in other parts of ermrsetjs we wanted use this prototype, we should call that function instead. I was looking at other ways of changing a constructor of a prototype to produce another type of prototype, but this method seemed more feasible.